geowebcache/rest/src/main/java/org/geowebcache/rest/service/FormService.java

```diff
@@ -182,28 +182,31 @@
             if (tl == null) {
                 throw new RestException("No layer specified", HttpStatus.BAD_REQUEST);
             }
-            handleDoSeedPost(params, tl);
+            return handleDoSeedPost(params, tl);
         } else {
```

Comment:
Technically, the else is now unnecessary. Probably clearer to just remove it and have the exception at top level.